daffodil-cli/src/it/scala/org/apache/daffodil/cliTest/TestLayers.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.daffodil.cliTest
+
+import org.junit.Test
+import org.apache.daffodil.CLI.Util._
+import org.apache.daffodil.Main.ExitCode
+
+class TestLayers {
+  @Test def test_Layer_Execution_Error(): Unit = {
+    val schema = path("daffodil-test/src/test/resources/org/apache/daffodil/layers/bad.dfdl.xsd")
+    val classPath = Seq(
+      path("daffodil-test/src/test/resources"),
+      path(s"daffodil-test/target/scala-${scalaVersionMajorMinor}/test-classes"),
+    )
```

Comment:
Tests that change the classpath requiring forking, and so need to be in `daffodil-cli/src/it/`. Building the CLI isn't done by default for tests in `daffodil-cil/src/test`, only for integration.

That said, if you put the bad layer transformer somewhere in daffodil-cli/ it should be on the classpah for these tests and you won't need to change modify the classpath and you won't need forking. That would be preferred since forking tests are much slower and should be avoided where possible.